services/web-app/pages/assets/[host]/[org]/[repo]/[library]/[ref]/[asset].js

```diff
@@ -69,15 +83,161 @@ const Asset = ({ libraryData }) => {
     }
   ]
 
+  const assetPath = breadcrumbItems[1].path
+
   const seo = {
     title: name,
     description
   }
 
+  const { sponsor } = assetData.params
+  const SponsorIcon = teams[sponsor] && teams[sponsor].icon
+
+  const AssetItemMeta = ({ properties }) => {
+    const renderStatus = () => {
+      const { name: assetStatus } = status[assetData.content.status]
+
+      if (!assetStatus) return null
+
+      return (
+        <>
+          <StatusIcon className={styles.metaIcon} status={assetData.content.status} />
+          <span className={styles.metaStatus}>{assetStatus}</span>
+        </>
+      )
+    }
+
+    return (
+      <ul className={styles.meta}>
+        {properties.map((prop, i) => (
+          <li key={i}>{prop === 'status' && renderStatus()}</li>
+        ))}
+      </ul>
+    )
+  }
+
   return (
     <>
       <PageHeader title={seo.title} pictogram={TextInput} />
       <PageBreadcrumb items={breadcrumbItems} />
+      <Dashboard className={styles.content}>
+        <Column className={dashboardStyles.column}>
+          <DashboardItem
+            aspectRatio={{ sm: '2x1', md: '1x1', lg: '3x4', xlg: '1x1' }}
+            border={['sm']}
+          >
+            <p className={dashboardStyles.title}>Library</p>
+            <h3 className={dashboardStyles.titleLarge}>{libraryData.content.name}</h3>
+            <Link href={`${assetPath}`}>
+              <a className={clsx(styles.metaLinkLarge, dashboardStyles.subcolumn)}>
```

Comment:
I'm seeing places where column and subcolumn are applied to elements other that `Column`s. Can you review these and remove when not needed?